test/test_nn.py

```diff
@@ -5759,6 +5845,70 @@ def multimarginloss_weights_no_reduce_test():
         check_eval=True,
         desc='3d_elementwise_affine_tracking_stats',
     ),
+    dict(
+        module_name='GroupNorm',
+        constructor_args=(3, 6, 1e-3, 0.3),
+        input_size=(4, 6, 5),
+        cudnn=True,
+        check_eval=True,
+        desc='1d_affine',
+    ),
+    dict(
+        module_name='GroupNorm',
+        constructor_args=(5, 5, 1e-3, 0.3, False),
+        input_size=(4, 5, 5),
+        cudnn=True,
+        check_eval=True,
+        desc='1d_no_affine_IN',
+    ),
+    dict(
+        module_name='GroupNorm',
+        constructor_args=(1, 5, 1e-3, 0.3, False),
+        input_size=(4, 5, 5),
+        cudnn=True,
+        check_eval=True,
+        desc='1d_no_affine_LN',
+    ),
+    dict(
+        module_name='GroupNorm',
+        constructor_args=(2, 6, 1e-3, 0.3, True, True),
+        input_size=(4, 6, 5),
+        cudnn=True,
+        check_eval=True,
+        desc='1d_affine_tracking_stats',
+    ),
+    dict(
+        module_name='GroupNorm',
+        constructor_args=(3, 6, 1e-3, 0.3),
+        input_size=(4, 6, 2, 3),
+        cudnn=True,
+        check_eval=True,
+        desc='2d_affine',
+    ),
+    dict(
+        module_name='GroupNorm',
+        constructor_args=(3, 3, 1e-3, 0.3, False),
+        input_size=(4, 3, 2, 3),
+        cudnn=True,
+        check_eval=True,
+        desc='2d_no_affine_IN',
+    ),
+    dict(
+        module_name='GroupNorm',
+        constructor_args=(1, 3, 1e-3, 0.3, False),
+        input_size=(4, 3, 2, 3),
+        cudnn=True,
+        check_eval=True,
+        desc='2d_no_affine_LN',
+    ),
+    dict(
+        module_name='GroupNorm',
+        constructor_args=(2, 4, 1e-3, 0.3, True, True),
+        input_size=(4, 4, 2, 3),
+        cudnn=True,
+        check_eval=True,
+        desc='2d_affine_tracking_stats',
+    ),
```

Comment:
Testing higher dimensional cases is not necessary as it goes through the same `.view(1, batch * group, -1)` reshape.